app/appmessage/rpc_get_blocks.go

```diff
@@ -18,7 +18,6 @@ func (msg *GetBlocksRequestMessage) Command() MessageCommand {
 func NewGetBlocksRequestMessage(lowHash string, includeBlockHexes bool, includeBlockVerboseData bool) *GetBlocksRequestMessage {
```

Comment:
Please update this signature. It should not contain includeBlockHexes and should contains includeTransactionVerboseData.